tests/core/test_api.py

```diff
@@ -63,6 +67,26 @@ def test_generate_partial_json_array(self):
         # TODO: Need test this
         pass
 
+    # Test the API is ready after the init process and is not ready when starting and during the init process
+    @patch("data_access_service.core.api.DataQuery.GetAodn")
```

Comment:
let say class API
def getStatus() -> is_ready

then you can mock this function to true or false

when true, if you hit /healthy you should get Http200 else should get something else